test/kixi/stats/distribution_test.cljc

```diff
@@ -102,6 +102,22 @@
                (every? nil? (vals summary))
                (every? number? (vals summary))))))))
 
+(defspec cdf-and-quantile-are-inverses
+  test-opts
+  (for-all [a gen/int
+            r gen-rate
+            s gen-shape
+            p gen-probability
+            alpha gen-pos-real
+            k (gen/fmap inc gen/nat)
+            d gen-small-n]
+    (let [dist (sut/normal {:location a :scale k})] (is (=ish p (sut/cdf dist (sut/quantile dist p)))))
+    (let [dist (sut/log-normal {:location a :scale k})] (is (=ish p (sut/cdf dist (sut/quantile dist p)))))
+    (let [dist (sut/cauchy {:location a :scale alpha})] (is (=ish p (sut/cdf dist (sut/quantile dist p)))))
+    ;; seems to div by 0 in quantile (let [dist (sut/t {:v d})] (is (=ish p (sut/cdf dist (sut/quantile dist p)))))
```

Comment:
This was a problem with p = 0.0 in `quantile-t`. I've pushed a fix to master, thank you for identifying an issue!

The `is` expression arguments are identical, so test failures can't easily be traced to a specific distribution. Perhaps you could rename `dist` uniquely for each assertion? Or perhaps create a helper function such as `(fn [dist p] (cdf dist (quantile dist p)))` so that the `let` is no longer required?

Reply:
Sounds good. I'll make a helper function

Reply:
I made the helper function and checked in the changes.  However, I am still seeing errors with the Student T.
I have currently commented out the test for Student T and Chi-squared.

> FAIL in (cdf-and-quantile-are-inverses) (distribution_test.cljc:121)
> expected: (=ish p (cdf-quantile (sut/t {:v d}) p))
>   actual: (not (=ish 0.0 nil))

Seems like the p = 0.0 is still the culprit.

Reply:
Forgot to mention that I updated this branch to account for the commit you made yesterday in master.